### ISSUE TITLE
Are users avoiding `quarkus update` it's because they don't know what it does or doesn't do?

### ISSUE THREAD

**Issue by rolfedh** [quarkusio/quarkus#32807]
### Describe the bug

In a couple of email threads, users updating to Quarkus 3 hint that they've opted to use the Migration guide instead of the `upgrade command` because they don't have enough information about what the command does. I've expressed this as the following user story:

### User story
. As a developer deciding whether to use the update command, knowing which tasks the update command handles makes it more likely that I'll use this feature. (Otherwise, I might avoid it.) 
. This way, I can use the command to do the heavy lifting and make any remaining updates manually.

### Expected behavior

I would like the command output and/or documentation to indicate what changes the command makes.
One possible way to do this might be to add a column or icon to each item in the following guides indicating whether it is handled by the update command/OpenRewrite recipe.
- https://github.com/quarkusio/quarkus/wiki/Migration-Guide-3.0
- https://github.com/quarkusio/quarkus/wiki/Migration-Guide-3.0:-Hibernate-ORM-5-to-6-migration  

### Actual behavior

There's no information about what the update command does.
As a user, my only option is to try it out and review a diff of my code.

**Comment by geoand** [comment]
cc @maxandersen

**Comment by gsmet** [comment]
> In a couple of email threads

Could you point them to me on Zulip? Thanks.

**Comment by rolfedh** [comment]
> Could you point them to me on Zulip? Thanks.

Hi @gsmet. For the first instance I had in mind, I've provided the reference to you directly on Zulip. Reviewing the other instance I had in mind, I realized that the user did not indicate avoiding the update command. I have forwarded this one to you directly.

**Comment by maxandersen** [comment]
Can you share the info here ? 

My assumption or expectation is that not all will like to use the update tool no matter how nice we describe it :)

**Comment by maxandersen** [comment]
Yeah; so that thread is an example from the kind of user I refer to. Experienced and been in quarkus ecosystem for a long time. Typically prefer to hand move things and it does not surprise me :) no amount of documentation would get him to use tool for it. 

If anything I would say we should highlight that the tool can be run to make a diff rather than change in place. Might calm someone who are concerned :)

**Comment by rolfedh** [comment]
@maxandersen and @gsmet, let me ask you to look at this another way:

Suppose that you are a developer whose goal is to completely update your projects to Quarkus 3.
How would you know when you've reached that goal?
